### PR TITLE
improve useUpdateAtom tests

### PR DESCRIPTION
**Pull request by Aslemammad**
@dai-shi  I'm improving the tests as #88 notes and I'll improve other tests soon like basic, dependency,...(I'm a bit busy now)

### CODE REVIEW

File: tests/utils/useUpdateAtom.test.tsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
-import React, { StrictMode, useEffect, useRef } from 'react'
```
Comment (dai-shi) [inline tests/utils/useUpdateAtom.test.tsx:1]: I would rather add `<StrictMode>` around `<Provider>`, if it doesn't break the test.

**Review by dai-shi — APPROVED** [review]
It's ok as is, but added a comment.

File: tests/utils/useUpdateAtom.test.tsx (hunk 1 above)
Comment (Aslemammad) [reply tests/utils/useUpdateAtom.test.tsx:1]: done

**Review by dai-shi — APPROVED** [review]
Cool.

**Comment by Aslemammad** [comment]
@dai-shi Thanks.

**Pull request merged**